config/environment.rb

```diff
@@ -4,6 +4,7 @@
 SITE_NAME = 'OpenStax Exchange'
 COPYRIGHT_HOLDER = 'Rice University'
 COPYRIGHT_START_YEAR = 2013
+DEFAULT_LOCK_TIMEOUT_SECS = 20
```

Comment:
I think maybe 10 seconds would be more suitable. I don't expect these routines to ever take more than 5. Except maybe when waiting for the locks themselves.